src/Hazelcast.Net/Core/EnumerableExtensions.cs

```diff
@@ -32,6 +32,20 @@ internal static class EnumerableExtensions
         public static IEnumerable<T> Shuffle<T>(this IEnumerable<T> source)
             => source.OrderBy(x => RandomProvider.Next());
 
+        /// <summary>
+        /// Shuffles an enumerable.
```

Comment:
Source is not just enumerable, is enumerable mention intended?

Reply:
bad copy/paste, fixing